### PR TITLE
Fixing long class name

### PR DESCRIPTION
**Pull request by adityashahms**
**Task name**: AzureTestPlanV0

**Description**: Fixing the parsing logic of FQN received from associated automation and finding against all the test cases found in the repo. 

**Documentation changes required:** (Y/N) N

**Added unit tests:** (Y/N) N

**Attached related issue:** (Y/N) N

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

File: _generated/AzureTestPlanV0_Node20/.npmrc
Comment (KathanS) [inline _generated/AzureTestPlanV0_Node20/.npmrc:1]: Should we remove changes from .npmrc and package-loc.json files?

File: Tasks/AzureTestPlanV0/Invokers/pythoninvoker.ts [hunk 1]
```diff
@@ -14,10 +14,32 @@ export async function executePythonTests(testsToBeExecuted: string[]):Promise<nu
...
+            tl.debug(`Test ${test} not found in discovered tests`);
+        }
+        else{
+            try{
+                testsToRun.push(testStringtoFQNMap.get(test));
+            }
+            catch (err){
+                console.log(err);
+                console.log(String(err));
+                testsToRun.concat([`'${testStringtoFQNMap.get(test)}'`]);
```
Comment (KathanS) [inline Tasks/AzureTestPlanV0/Invokers/pythoninvoker.ts:36]: Should we remove concat?

File: _generated/AzureTestPlanV0_Node20/.npmrc
Comment (adityashahms) [reply _generated/AzureTestPlanV0_Node20/.npmrc:1]: .npmrc are auto-generated and is in line with the npmrc file of the Node 16 task. For the package-lock.json file, I am reverting the changes.

File: Tasks/AzureTestPlanV0/Invokers/pythoninvoker.ts (hunk 1 above)
Comment (adityashahms) [reply Tasks/AzureTestPlanV0/Invokers/pythoninvoker.ts:36]: Good Catch. Let me fix it.

Comment (adityashahms) [reply Tasks/AzureTestPlanV0/Invokers/pythoninvoker.ts]: Fixed

File: _generated/AzureTestPlanV0_Node20/.npmrc
Comment (adityashahms) [reply _generated/AzureTestPlanV0_Node20/.npmrc:1]: We have to make changes to package-lock.json file, we were referencing public npm registry feed but with SFI, we have to now use any public packages from our private npm package feed. Hence made the changes.

**Review by KathanS — APPROVED** [review]

**Pull request merged**